src/assignments/flask-game.rst

```diff
@@ -0,0 +1,6 @@
+.. _flask-game:
+
+Assignment #6: Flask Game
+=========================
+
+Lorem ipsum...
```

Comment:
Assignment #6 was already merged, will this cause a problem?

Reply:
Nope!  We're good.